fivetran_test.go

```diff
@@ -515,6 +516,30 @@ func cleanupWebhooks() {
 	}
 }
 
+func cleanupTeams(nextCursor string) {
+	svc := Client.NewTeamsList()
+
+	if nextCursor != "" {
+		svc.Cursor(nextCursor)
+	}
```

Comment:
Iteratively this will work, but it's not reliable.
Once you've deleted first page - this cursor might be invalidated.
Just ignore cursor at all - fetch first page on every iteration and delete - it is easier and more reliable.